resources/assets/less/bem/landing-footer.less

```diff
@@ -0,0 +1,159 @@
+/**
+ *    Copyright 2015 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+ .landing-footer {
+    height: 100%;
+    flex-grow: 1;
+    .at2x("/images/backgrounds/page-000.png", 500px, 500px);
+}
+
+.landing-sitemap {
+    position: relative;
+    display: flex;
+
+    &.osu-layout__row {
+        max-width: 800px;
+        margin: 35px 0 40px;
+
+        @media @mobile {

```

Comment:
whenever possible, scale up (mobile first, changes with `@media`'s `min-size`/`@desktop`) instead of scale down.